### ISSUE TITLE
Feedback from group Inaugural-Project-2021_BSJH

### ISSUE THREAD

**Issue by Sehghee** [NumEconCopenhagen/projects-2021-ska#2]

**Comment by Sehghee** [comment]
1. The best part of the projecct was:

2. The hardest part of the project to understand was:

3. This part of the project could be better documented:

4, An idea for an improvement/clarification could be:

5. An idea for an extension could be:

**Comment by jacoropke** [comment]
I agree with Sehghee's points above(same group) but have a few extras. Your code is easy to understand and follow.

Regarding improvements:
you should remove the import dataproject from the first cell if there is no external module. I could run the notebook without, so you should delete this line. If there is, in fact, an external module then it should be uploaded to the repository to allows others to run it. 

If you wanted to extend the project you could perhaps include explanatory factors as to explain differences in unemployment. You could also attempt to compute volatility measures(standard deviation, variance) or similar to compare volatility across series.
